### DISCUSSION TITLE
hello how I can make my own klipperscreen theme

### DISCUSSION THREAD

**Discussion by taktojakuba** [KlipperScreen/KlipperScreen#1127] [Q&A]
or where I can get one and how to install it

**Comment by bumbeng** [comment]
this is my theme https://github.com/bumbeng/Klipperscreen_blurry_theme.
I have copy an existing Folder in /klipperscreen/styles and rename it to my design name. I changed background and the style.css. As reference i used the base.css in styles folder. ![image](https://github.com/KlipperScreen/KlipperScreen/assets/111509593/4395d3ea-0fe3-4e6f-b052-6ea89fb17459)

**Comment by alfrix** [comment]
https://klipperscreen.readthedocs.io/en/latest/Theming/
